l10n_it_intrastat/models/account.py

```diff
@@ -310,13 +316,18 @@ def _onchange_partner_id(self):
     @api.model_create_multi
     def create(self, vals_list):
         for val in vals_list:
-            if "intrastat" not in val and "fiscal_position_id" in val:
-                intrastat = (
-                    self.env["account.fiscal.position"]
-                    .browse(val["fiscal_position_id"])
-                    .intrastat
+            if "fiscal_position_id" in val:
+                fiscal_position_id = self.env["account.fiscal.position"].browse(
```

Comment:
Potrebbe essere utile
```suggestion
                fiscal_position = self.env["account.fiscal.position"].browse(
```
Per distinguere che le variabili con `_id` sono degli interi, cosa ne pensi?

Reply:
Vero